test/spec/services/basket.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+
+describe('Service: Basket', function () {
+
+  // load the service's module
+  beforeEach(module('tmaClientApp'));
+
+  // instantiate service
+  var mockBasket;
+
+  beforeEach(inject(function(Basket) {
+    mockBasket = Basket;
+  }));
+
+  it('should initially be empty', function(){
+    expect(mockBasket.count()).toBe(0);
+  });
+
+  describe('Basket.get()', function() {
+    it('should have a get function', function() {
+      expect(angular.isFunction(mockBasket.get)).toBe(true);
```

Comment:
it is better to describe what the `get` does than whether it exists, so again, something like:

```javascript
expect(mockBasket.get()).toEqual(...)
```